topics/qodana-github-application.md

```diff
@@ -3,6 +3,10 @@
 
 ![Qodana GitHub App](qodana-app-banner.png)
 
+<note>
+<include src="lib_qd.xml" include-id="github-app-supported-techs"/>
```

Comment:
Thank you! 👍
A small markup tweak, otherwise lgtm.

```suggestion
<p>
    <include src="lib_qd.xml" include-id="github-app-supported-techs"/>
</p>
```